ci/blackbox/rdsbroker_test.go

```diff
@@ -306,50 +305,77 @@ func openConnection(databaseURI string) (*sql.DB, error) {
 	return sql.Open(dbURL.Scheme, dsn)
 }
 
-func setupPermissionsTest(databaseURI string) error {
+func setupPermissionsTest(databaseURI, serviceID string) error {
 	db, err := openConnection(databaseURI)
 	if err != nil {
 		return err
 	}
 	defer db.Close()
 
-	_, err = db.Exec("CREATE TABLE " + permissionCheckTableName + " (id integer)")
+	_, err = db.Exec("CREATE TABLE permissions_check (id integer)")
 	if err != nil {
 		return fmt.Errorf("Error creating table: %s", err.Error())
 	}
 
-	_, err = db.Exec("INSERT INTO " + permissionCheckTableName + " VALUES(42)")
+	_, err = db.Exec("INSERT INTO permissions_check VALUES(42)")
 	if err != nil {
 		return fmt.Errorf("Error inserting record: %s", err.Error())
 	}
 
+	switch serviceID {
```

Comment:
Done, but I haven't tested it in any way. It's very late on a Friday afternoon, I'll sanity check it on Monday.